### PR TITLE
Added absolute paths to navigation links

### LINKED ISSUE #79: When routing from a `/skills/{skill_id}` page to `Team` the `team` path should not be added to the url
`http://ec2-54-147-37-162.compute-1.amazonaws.com:8000/skills/team`

### PR DESCRIPTION
**Pull request by dane-johnson**
Closes #79

### CODE REVIEW

**Review by solkaz — APPROVED** [review]

**Review by solkaz — DISMISSED** [review]
Could we add tests to ensure that something like this doesn't come up again?

File: src/Header.jsx [hunk 1]
```diff
@@ -14,10 +14,10 @@ const Header = ({ children }) => (
       </Navbar.Header>
         <Nav>
           <NavItem eventKey={1}>
-            <Link to="skills">Skills</Link>
```
Comment (thebho) [inline src/Header.jsx:17]: Add a comment to explain the extra `/`

**Review by thebho — CHANGES_REQUESTED** [review]

File: src/Header.jsx [hunk 2]
```diff
@@ -12,12 +12,14 @@ const Header = ({ children }) => (
           <Link to="/">Skill Directory</Link>
         </Navbar.Brand>
       </Navbar.Header>
+      //Make sure all these paths are absolute (starting with a '/') so the
+      //route will be handled properly
```
Comment (dane-johnson) [inline src/Header.jsx:17]: Added this comment

**Comment by dane-johnson** [comment]
Don't merge this yet. I think this comment is not being escaped properly.

**Review by thebho — APPROVED** [review]

**Pull request merged**